_data/tool_and_resource_list.yml

```
@@ -1290,3 +1290,407 @@
+  url: https://cran.r-project.org/web/packages/validate/validate.pdf
+- description: Validate dataset by columns and rows using convenient predicates inspired by 'assertr' package.
+  id: data.validator
+  name: data.validator
+  url: https://cran.r-project.org/web/packages/data.validator/data.validator.pdf
+- description: A collection of some tests commonly used for identifying outliers.
+  id: outliers
+  name: outliers
+  url: https://cran.r-project.org/web/packages/outliers/outliers.pdf
+- description: A comprehensive but easy-to-use Python library for detecting anomalies in multivariate data.
+  id: pyod
+  name: pyod
+  url: https://pyod.readthedocs.io/en/latest/
+- description: Multiple imputation using Fully Conditional Specification (FCS) implemented by the MICE algorithm as described in Van Buuren and Groothuis-Oudshoorn (2011) <doi:10.18637/jss.v045.i03>.
+  id: mice
+  name: mice
+  url: https://cran.r-project.org/web/packages/mice/mice.pdf
+- description: Supply functions for the creation and handling of missing data as well as tools to evaluate missing data methods
+  id: missmethods
+  name: missMethods
+  url: https://cran.r-project.org/web/packages/missMethods/missMethods.pdf
+- description: A tool designed to provide ultrafast all-in-one preprocessing and quality control for FastQ data.
+  id: fastp
+  name: fastp
+  url: https://github.com/OpenGene/fastp
+- description: Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.
+  id: cutadapt 
+  name: Cutadapt
+  url: https://cutadapt.readthedocs.io/en/stable/
+- description: A novel analysis tool developed for quality control and preprocessing of FASTQ and SAM/BAM data.
+  id: soapnuke 
+  name: SOAPnuke
+  url: https://github.com/BGI-flexlab/SOAPnuke
+- description: Guppy is a bioinformatics toolkit that enables real-time basecalling and several post-processing features that works on Oxford Nanopore Technologies™ sequencing platforms.
+  id: guppy
+  name: Guppy
+  url: https://nanoporetech.com/es/document/Guppy-protocol
+- description: Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
+  id: dorado
+  name: Dorado
+  url: https://github.com/nanoporetech/dorado
+- description: NanoPlot is a plotting tool for long read sequencing data and alignments.
+  id: nanoplot
+  name: NanoPlot
+  url: https://github.com/wdecoster/NanoPlot
+- description: nanoq is an ultra-fast quality control and summary reports for nanopore reads
+  id: nanoq
+  name: nanoq
+  url: https://github.com/esteinig/nanoq
+- description: PycoQC computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data
+  id: pycoqc
+  name: PycoQC
+  url: https://a-slide.github.io/pycoQC/
+- description: LongQC is a tool for the data quality control of the PacBio and ONT long reads, and it has two functionalities: sample qc and platform qc.
```

```suggestion
- description: "LongQC is a tool for the data quality control of the PacBio and ONT long reads, and it has two functionalities: sample qc and platform qc."
```

A value can not have a colon in yaml, otherwhise it is detected as another key.